molo/core/templates/patterns/basics/banners/banner_default.html

```diff
@@ -1,24 +1,27 @@
-{% block banner %}
-  {% load wagtailcore_tags wagtailimages_tags i18n %}
-
-  {% if bannerpages %}
-  {% for bannerpage in bannerpages %}
-    <div class="banner">
-      {% if bannerpage.banner %}
-    	    {% if bannerpage.banner_link_page %}
-    	       <a href="{% pageurl bannerpage.banner_link_page %}" target="_top" class="banner__item-link">
-               {% image bannerpage.banner width-320 %}
-             </a>
-    	    {% elif bannerpage.external_link %}
-    	       <a href="{{ bannerpage.external_link }}" target="_blank" class="banner__item-link">
-               {% image bannerpage.banner width-320 %}
-             </a>
-    	    {% else %}
-    	       {% image bannerpage.banner width-320 %}
-    	    {% endif %}
-      	{% endif %}
-    </div>
-  {% endfor%}
+{% load wagtailcore_tags wagtailimages_tags i18n %}
+<div class="banner">
+  {% if bannerpage.banner %}
```

Comment:
@codiebeulaine with having the textbanner field, we can use that logic to differentiate the too type of banner here